index.md

```diff
@@ -182,7 +182,7 @@ message Person {
 
 ``` [2]
 # github.com/protocolbuffers/protoscope
-1: {"Miguel Young de la Sota"}
+1: {"Matthew Kulukundis"}
```

Comment:
`Matthew Fowles Kulukundis` according to the binary fragment ;)

Reply:
Whoops :P